yarn.lock

```diff
@@ -12341,6 +12359,32 @@ __metadata:
   languageName: node
   linkType: hard
 
+"cpy-cli@npm:^6.0.0":
```

Comment:
Hey @lightning-sagar, the `yarn.lock` file remained in the previous state.
Can you please restore it too?